src/Nethermind/Nethermind.JsonRpc/WebSockets/JsonRpcWebSocketsClient.cs

```diff
@@ -57,7 +57,7 @@ string SerializeTimeoutException()
             }
 
             Stopwatch stopwatch = Stopwatch.StartNew();
-            using JsonRpcResult result = await _jsonRpcProcessor.ProcessAsync(Encoding.UTF8.GetString(data.ToArray()));
+            using JsonRpcResult result = await _jsonRpcProcessor.ProcessAsync(Encoding.UTF8.GetString(data.Span));
 
             string resultData;
```

Comment:
It would be great if  we didn't have to allocate this as string and just sent response while serializing it, like in http mode